gateway-service-impl/src/main/java/org/hypertrace/gateway/service/common/util/MetricAggregationFunctionUtil.java

```diff
@@ -113,4 +116,27 @@ public static AttributeKind getValueTypeFromFunction(
         return metadata.getValueKind();
     }
   }
+
+  public static Value getValueFromFunction(
```

Comment:
Also this can replace RequestHandler.getValueForFunctionExpression() in the explore impl

Reply:
I have searched for RequestHandler.getValueForFunctionExpression() in gateway Service couldn't find anything

Reply:
https://github.com/hypertrace/gateway-service/blob/cf27396552efd5e12383ccd7e1079997beda81e1/gateway-service-impl/src/main/java/org/hypertrace/gateway/service/explore/RequestHandler.java#L296

Reply:
Sorry my bad. Fixed this can you please check now.